### PR TITLE
Prodtest versioning

### PR DESCRIPTION
**Pull request by matejcik**

### CODE REVIEW

File: core/embed/prodtest/towncrier.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../tools/towncrier.toml
```
Comment (mmilata) [inline core/embed/prodtest/towncrier.toml:1]: Dead link?

File: core/embed/prodtest/version.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #define VERSION_MAJOR 0
 #define VERSION_MINOR 2
-#define VERSION_PATCH 2
+#define VERSION_PATCH 4
```
Comment (mmilata) [inline core/embed/prodtest/version.h:3]: Why skip 3?

File: core/embed/prodtest/towncrier.toml (hunk 1 above)
Comment (matejcik) [reply core/embed/prodtest/towncrier.toml:1]: it's not dead locally :woman_shrugging: i think this is github bug. `core/ ( embed/ ( prodtest / .. ) /.. ) /.. / tools/towncrier.toml` exists

File: core/embed/prodtest/version.h (hunk 2 above)
Comment (matejcik) [reply core/embed/prodtest/version.h:3]: it's there in the commits actually :) first go to 0.2.3 for a commit that will be tagged, and bump immediately after in "bump version after release"

**Review by mmilata — APPROVED** [review]

**Pull request merged**